docs/pt-br/SUMMARY.md

```diff
@@ -0,0 +1,21 @@
+# Vuex
+
+> Nota: para os usu�rios TypeScript, vuex@3.0+ requer vue@2.5+ e vice-versa.
+
+- [Notas de vers�o] (https://github.com/vuejs/vuex/releases)
```

Comment:
Can we remove the space between `]` and `(`? It's not rendered as link on generated docs (You can confirm it by `npm i -g gitbook-cli && npm run docs` on local)

Reply:
Thank you  so much for the Quick Tip. I go fix it right now.

Reply:
Fixed. If have something more please tell me. Thanks